Automattic/wp-calypso
# Pricing: Recognize Boost and Social as included in Jetpack Complete

This PR adds Boost and Social to the list of products included in Jetpack Complete, so that when people with an active Complete subscription visit the pricing page, they see "Manage Subscription" beside these products instead of "Add to cart."

Resolves #74573.

## Proposed Changes

* Add new feature slugs for Boost and both Social products (yearly and monthly terms).
* Add Boost and Social Advanced feature slugs to the list of included features for Jetpack Complete.
* Add the Social Basic feature to the list of "inferior" features for Jetpack Complete.
* Add logic on the pricing page so that when Social is included in a plan, we link visitors to the correct subscription management page.

## Testing Instructions

1. While logged in and with a site selected, visit the pricing page on Jetpack Cloud (or Calypso Blue).
2. If your site has an active Complete subscription, verify you see "Manage Subscription" buttons next to Boost and Social. Also check that these buttons link to the subscription management page for your Complete purchase.
3. If your site does not have an active Social or Boost subscription, a plan that does not include them, or no plan at all, verify the links beside Social and Boost say "Add to cart" and act exactly as they do in production: Boost goes to checkout, Social opens a modal with more info.

### Before / after

<img height="185" alt="Screenshot 2023-03-17 at 08 21 11" src="https://user-images.githubusercontent.com/670067/225919043-e9f398d7-7b2b-48a1-b714-8f9bae012781.png"> <img height="185" alt="Screenshot 2023-03-17 at 08 21 04" src="https://user-images.githubusercontent.com/670067/225919049-6127ab30-d034-4305-809f-81624094aee8.png">


## Pre-merge Checklist
